src/status.js

```diff
@@ -0,0 +1,12 @@
+const object = {
+  running: true,
+};
+
+function objectReturn() {

```

Comment:
It is a best practice to name the function using a verb as prefix to tell what it is doing

Reply:
Ok.